openslides/motions/config_variables.py

```diff
@@ -39,6 +39,15 @@ def get_config_variables():
         subgroup='General')
 
     yield ConfigVariable(
+        name='motions_recommendations_by',
+        default_value='Recommendation committee',
+        label='Name of recommendation committee',
+        weight=312,
+        group='Motions',
+        subgroup='General',
+        translatable=True)
```

Comment:
Ok. Recommendation section is now hidden if config value is empty. Help text added. The other "feature" looks a little bit overkill.